dask/bytes/tests/test_local.py

```diff
@@ -0,0 +1,214 @@
+from __future__ import print_function, division, absolute_import
+
+from time import sleep, time
+
+import pytest
+from toolz import concat, valmap, partial
+
+from dask import compute, get
+from dask.bytes.local import read_bytes, open_files, getsize
+from dask.bytes.core import open_text_files
+from dask.compatibility import FileNotFoundError
+from dask.utils import filetexts
+from dask.bytes import compression
+
+compute = partial(compute, get=get)
+
+# These get mirrored on s3://distributed-test/
```

Comment:
Not relevant here - and maybe this data should be stored in only a single place in the code?